addons/sale_pdf_quote_builder/models/sale_order.py

```diff
@@ -77,6 +78,14 @@ def _onchange_sale_order_template_id(self):
 
     # === ACTION METHODS === #
 
+    def action_confirm(self):
+        ICP = self.env['ir.config_parameter'].sudo()
+        if not ICP.get_param('sale.always_include_selected_documents'):
+            # Remove selected quotation documents on order confirmation
+            self.quotation_document_ids = [Command.clear()]
+            self.order_line.product_document_ids = [Command.clear()]
+        return super().action_confirm()
```

Comment:
Honestly not sure this is still needed with the config param 🤔.
If they say they wanna keep the documents on confirmed orders pdfs, the documents can stay on confirmation IMHO.